### ISSUE TITLE
URGENT: Upgrade Sentry Next.js SDK to version 7.77.0 or higher

### ISSUE THREAD

**Issue by taariq** [VolumeFi/paloma#956]
TL;DR
Please update the Sentry Next.js SDK to version 7.77.0 or higher immediately to mitigate a security vulnerability.

Summary
You are receiving this email because you are listed as the owner of the Sentry organization volume-x3. This organization appears to be using Sentry’s Next.js SDK, which has a critical vulnerability.

We fixed this vulnerability in the Sentry Next.js SDK version 7.77.0. We have no evidence of exploitation of this vulnerability at this time.

Please update the Sentry Next.js SDK to version 7.77.0 or higher immediately. If upgrading the SDK is currently not possible for you, removing the “tunnelRoute” option from the Sentry SDK configuration (usually located in the next.config.js or next.config.mjs files) will mitigate the vulnerability.

If you are hosting your application on Vercel, the exploit has been mitigated at the infrastructure level. However, we still recommend updating your SDK version to 7.77.0 or higher.

More details, including a detailed summary; timeline; and investigation techniques, will be available with the public release of a security advisory, currently scheduled for Thursday, November 9th, 2023 at 12:00 PM Pacific Time. This advisory will be published on the public GitHub for the Sentry Next.js SDK, available at [https://github.com/getsentry/sentry-javascript/security](https://go.sentry.marketing/Nzc2LU1KTi01MDEAAAGPNi1EZRzwC_YNA9ZnGwVBQstj_NeTiFYPVLEjLEGLIcgmUzU6PWtz2y2687rkvatDR2ZL98U=).

We are not releasing more details at this time to give you time to upgrade to the new SDK before the vulnerability becomes public.

**Comment by taariq** [comment]
@rootedbox is this complete?

**Comment by verabehr** [comment]
update is missing from the following @rootedbox : 
Curve momentum: https://github.com/VolumeFi/curve-momentum-bot-script/blob/master/package.json
Gecko Bot: https://github.com/VolumeFi/gecko-bot-script/blob/master/package.json
Gecko New Token: https://github.com/VolumeFi/gecko-newtoken-bot-script/blob/master/package.json
Stop loss: https://github.com/VolumeFi/stop-loss-bot-script/blob/master/package.json
Curve Stop loss: https://github.com/VolumeFi/curve-stop-loss-bot-script/blob/master/package-lock.json
Curve Lending: https://github.com/VolumeFi/curve-lending-bot/blob/main/package-lock.json

**Comment by rootedbox** [comment]
those have been upgraded on the server and changes pushed to GitHub
